### PR TITLE
Assignment 1 - Nathan Graham (@grahamna)

### PR DESCRIPTION
**Pull request by grahamna**
https://nice-forest-07f8c5e1e.3.azurestaticapps.net/

@grahamna

ExtraCredit Idea:
Added EWU .svg logo!

Still not sure if I'm doing this right...
Am I supposed to try to pull/merge from my main -> their main?
Or am I supposed to try to pull/merge from my Assignment1 -> their Assignment1?

Will be paired with @BenKnoll3 for future assignments

### CODE REVIEW

**Review by jdriscoll1 — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
One point per idea up to three points

Looks great! Good job on using Vue's features to make the Wordle game look as good as it does!

**Review by BenKnoll3 — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

All requirements are met. The wordle page is looking great!

File: wordle-web/src/views/game.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+
+
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:2]:
```suggestion
```

File: wordle-web/src/views/game.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          <RouterLink to="/">Home</RouterLink>
+        </v-btn>
+      </VCardActions>
+    </VCard>
+  </div>
+</template>
+
+
+
+
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:46]:
```suggestion

```

File: wordle-web/src/views/game.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      </VCardActions>
+    </VCard>
+  </div>
+</template>
+
+
+
+  
+  <style>
+    @media (min-width: 1024px) {
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:48]: I don't think this is doing anything.

File: wordle-web/src/views/game.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        label="WordGuess"
+        clearable
+        
+      ></v-combobox>
+
+      <VCardActions>
+        <VSpacer></VSpacer>
+        <v-btn variant="tonal">
+          <RouterLink to="/">Home</RouterLink>
+        </v-btn>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:38]:
```suggestion
        <v-btn variant="tonal" to="/">
          Home
        </v-btn>
```

File: wordle-web/src/views/game.vue [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+              :class="['d-flex justify-center align-center', ]"
+            >
+              <div>{{  }}</div>
+            </v-card>
+          </v-col>
+        </v-row>
+      </v-container>
+
+      <v-combobox
+
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:28]:
```suggestion
```

File: wordle-web/src/views/game.vue [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            </v-card>
+          </v-col>
+        </v-row>
+      </v-container>
+
+      <v-combobox
+      
+        label="WordGuess"
+        clearable
+
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:31]:
```suggestion
```

File: wordle-web/src/views/game.vue [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:1]: This file should be PascalCase. `Game.vue`

File: wordle-web/src/views/game.vue [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+<template>
+  <div class="game">
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:4]: I don't think you need this div

File: wordle-web/src/views/game.vue [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    <VCard variant="tonal">
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:5]:
```suggestion
    <v-card variant="tonal">
```
Be consistent in how you reference the Vuetify components. Usually, we use a dash for the Vuetify components and pascal for our components to easily distinguish between our stuff and not our stuff.

File: wordle-web/src/views/game.vue [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+      <VCardTitle>
+        WordleCopy
+      </VCardTitle>
+      <VCardSubtitle>
+        Sample Text. Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam, fuga consectetur.
+      </VCardSubtitle>
+      <v-divider :thickness="5"></v-divider>
+      <v-container>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:13]: Typically, containers are only used at the page level to control page whitespace on different sized screens.

File: wordle-web/src/views/game.vue [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+        <v-row justify="space-evenly" v-for="i in 6" :key="i">
+          <v-col v-for="j in 5" :key="`${i}-${j}`" cols="auto">
+            <v-card
+              height="50"
+              width="50"
+              :class="['d-flex justify-center align-center', ]"
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:19]:
```suggestion
              class="d-flex justify-center align-center"
```

File: wordle-web/src/views/game.vue [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+            >
+              <div>{{  }}</div>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:21]:
```suggestion
```

**Review by mmwoodfo — COMMENTED** [review]
Looks good, just make sure you produce clean code. You have alot of whitespace/unused stuff.

**Pull request closed without merging**